packages/core/src/config-loader/config-loader.ts

```diff
@@ -99,10 +99,18 @@ export class ConfigLoader {
   }
 
   async reload() {
-    // make sure everything is initialized
-    await this.isReady;
+    if (!this.workspaceInfo) await this.finishInit();
```

Comment:
had to rework the init flow, since we have the new `-b/--built` flag which affects the initial load. This isn't quite right but seems ok for now.